src/i18n/es.js

```diff
@@ -144,38 +135,22 @@ export default {
-        usuarios en tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos
-        visitantes poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda?
-        <a target="_blank" href="${urlHelpAdvancedReports}" ${patchForBlank}>HELP</a>.
-      </p>
-      `,
+    allow_enable_trial_HTML_MD: `Activa el periodo de prueba y accede a Reportes detallados sobre el comportamiento de los
+    usuarios en tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos
+    visitantes poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda? [HELP](${urlHelpAdvancedReports}).`,
     allow_enable_trial_button: `Activa período de prueba`,
     allow_enable_trial_title: `Prueba Automation de Comportamiento en Sitio por tiempo limitado`,
     datahub_not_domains_title: `Agrega el dominio de tu Web y analiza el comportamiento de tus usuarios`,
-    no_domains_HTML: `
-      <p>
-        Registra el o los dominios sobre los que quieres realizar el seguimiento y accede a Reportes
-        detallados. Descubre cuáles son las páginas más visitadas de tu Sitio Web o E-commerce, cuántos
-        visitantes poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda?
-        <a target="_blank" href="${urlHelpAdvancedReports}" ${patchForBlank}>HELP</a>.
-      </p>
-      `,
+    no_domains_HTML_MD: `Registra el o los dominios sobre los que quieres realizar el seguimiento y accede a Reportes
+    detallados. Descubre cuáles son las páginas más visitadas de tu Sitio Web o E-commerce, cuántos
+    visitantes poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda? [HELP](${urlHelpAdvancedReports}).`,
     no_domains_button: `Agrega tu dominio`,
     no_domains_button_destination: `${urlSiteTracking}`,
-    upgrade_account_free_HTML: `
-      <p>
-        Contrata un Plan Pago y accede a Reportes detallados sobre el comportamiento de los usuarios en
-        tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos visitantes
-        poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda? Presiona
-        <a target="_blank" href="${urlHelpAdvancedReports}" ${patchForBlank}>HELP</a>.
-      </p>
-      <p>
-        Contratando cualquier Plan Pago podrás utilizar esta funcionalidad de forma bonificada por tiempo
-        limitado. <a href="${urlBuyMonthly}">COMPRA AHORA</a>.
-      </p>
+    upgrade_account_free_HTML_MD: `Contrata un Plan Pago y accede a Reportes detallados sobre el comportamiento de los usuarios en
+    tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos visitantes
+    poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda? Presiona [HELP](${urlHelpAdvancedReports}).
+
+Contratando cualquier Plan Pago podrás utilizar esta funcionalidad de forma bonificada por tiempo limitado. [COMPRA AHORA](${urlBuyMonthly}).
       `,
```

Comment:
I think that this way to indent could be error-prone. Maybe we could remove all indentations, and maybe also adding an interline at the beginning:

```suggestion
    upgrade_account_free_HTML_MD: `
Contrata un Plan Pago y accede a Reportes detallados sobre el comportamiento de los usuarios en
tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos visitantes
poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda? Presiona [HELP](${urlHelpAdvancedReports}).

Contratando cualquier Plan Pago podrás utilizar esta funcionalidad de forma bonificada por tiempo limitado. [COMPRA AHORA](${urlBuyMonthly}).
`,
```

Reply:
Could it work?

Reply:
yes i updated